gitlab.yml

```diff
@@ -0,0 +1,23 @@
+provider:
+  name: faas
+  gateway: http://127.0.0.1:8080
+
+functions:
+  system-gitlab-event:
+    lang: go
+    handler: ./gitlab-event
+    image: martindekov/gitlab-event:0.0.1
+    labels:
+      openfaas-cloud: "1"
+      role: openfaas-system
+    environment:
```

Comment:
We need to add `content_type: text/plain` otherwise GitLab continually re-sends the message since it can't parse the response as JSON. cc @ivanayov @bartsmykla